app/controllers/recipe_foods_controller.rb

```diff
@@ -0,0 +1,43 @@
+class RecipeFoodsController < ApplicationController
+  def new
+    @recipe_food = RecipeFood.new
+  end
+
+  def edit
+    @recipe_food = RecipeFood.find(params[:id])
+  end
+
+  def create
+    @recipe_food = RecipeFood.new(recipe_food_params)
+    @recipe_food.recipe_id = params[:recipe_id]
+    if @recipe_food.save
+      flash[:success] = 'Ingredient added successfully!'
+      redirect_to recipe_path(params[:recipe_id])
+    else
+      flash.now[:error] = 'Error: ingredient could not be added!'
+      render :new, locals: { recipe_food: @recipe_food }
+    end
+  end
+
+  def destroy
+    @recipe_food = RecipeFood.find(params[:id])
+    @recipe_food.destroy!
+    flash[:success] = 'Ingredient was deleted successfully!'
+    redirect_to recipe_path(@recipe_food.recipe_id)
+  end
```

Comment:
- [ ] @Team it is good practice to create success and fail paths when you create methods such as destroy, this enables easier troubleshooting in case of an error.